### PR TITLE
Moved Windows setup deployment to AppVeyor

### PR DESCRIPTION
**Pull request by Mailaender**
as we're using Travis' Docker infrastructure #7311 to avoid the nuget timeouts, but in those containers, we can't install packages properly, so the build of the packages fails.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
   project: OpenRA.sln
   verbosity: minimal
+
 test_script:
 - nunit-console-x86.exe OpenRA.Test.dll
-deploy: off
\ No newline at end of file
+
+after_build:
+- '%NSIS_ROOT%\makensis /DSRCDIR="." /DDEPSDIR="./thirdparty/windows"'
```
Comment (obrakmann) [inline appveyor.yml]: I suppose this is missing the path to the openra.nsi file

File: appveyor.yml [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
 test_script:
 - nunit-console-x86.exe OpenRA.Test.dll
-deploy: off
\ No newline at end of file
+
+after_build:
+- cd packaging/windows
+- '%NSIS_ROOT%\makensis /DSRCDIR="." /DDEPSDIR="./thirdparty/windows"' OpenRA.nsi
+- cd ..
+- move build/OpenRA-*.exe %APPVEYOR_BUILD_FOLDER%
```
Comment (obrakmann) [inline appveyor.yml]: wrong indentation? https://ci.appveyor.com/project/OpenRA/openra/build/1.0.630

Comment (Mailaender) [inline appveyor.yml]: misplaced `'`

File: appveyor.yml [hunk 3]
```diff
@@ -1,9 +1,32 @@
...
+
 test_script:
-- nunit-console-x86.exe OpenRA.Test.dll
-deploy: off
\ No newline at end of file
+  - nunit-console-x86.exe OpenRA.Test.dll
+
+after_build:
+  - cd packaging/windows
+  - '%NSIS_ROOT%\makensis /DSRCDIR="." /DDEPSDIR="./thirdparty/windows" OpenRA.nsi'
```
Comment (obrakmann) [inline appveyor.yml]: if you cd into `packing/windows`, the SRCDIR and DEPSDIR paths need to be adjusted. Or you don't move down into that directory and give the full path to OpenRA.nsi instead.

**Comment by reaperrr** [comment]
You might have noticed already, but AppVeyor build failed:
> File: "C:\projects\openra\DOCUMENTATION.md" -> no files found.

**Comment by Mailaender** [comment]
Not the first time today. I am learning PowerShell the hard way.

**Comment by pchote** [comment]
Can we move the package creation step to `after_test:`?  We should aim to fail as early as possible in the case of errors.

**Comment by Mailaender** [comment]
Sure, updated.

**Comment by Mailaender** [comment]
We can also think about moving this to `before_deploy:` later.

**Pull request merged**